### DISCUSSION TITLE
<img> support for version 2.0 or higher

### DISCUSSION THREAD

**Discussion by pritpatek** [dompdf/dompdf#3274] [General]
I want to include a .png image using `<img>`. Currently this is not happening and I wanted to see if people have answers. I am using Php 8.2. I have tried solutions, all of which can be found in this Stackoverflow discussion:

https://stackoverflow.com/questions/25558449/dompdf-image-not-readable-or-empty

These solutions didn't work for me.

**Comment by bsweeney** [comment]
You don't indicate if your image is local or remote. Resource references may be restricted if you have not configured your Options to allow access.

Recommended reading: [Resource References and URI Validation](https://github.com/dompdf/dompdf/wiki/Usage#resource-references-and-uri-validation)
